### PR TITLE
Remove nashorn

### PR DESCRIPTION
**Pull request by linusg**
Nashorn has not seen any development activity in the last three years, its score has been unchanged throughout the lifetime of test262.fyi until dropping to less than 10% recently ~~for unknown reasons~~ (probably the timeout tweaks), and it takes up 3h of runner time during every run. Not worth keeping :)

image

image

image

### CODE REVIEW

File: scripts/downloadOldResults.mjs [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import { writeFileSync, mkdirSync, existsSync } from 'fs';
 
-const engines = [ 'v8', 'sm', 'jsc', 'chakra', 'graaljs', 'hermes', 'kiesel', 'libjs', 'engine262', 'qjs', 'xs', 'nashorn', 'rhino', 'babel', 'swc' ];
```
Comment (linusg) [inline scripts/downloadOldResults.mjs:3]: This is unused within the file

File: site/index.html [hunk 2]
```diff
@@ -135,12 +132,12 @@ <h2>daily runner of <a href="https://github.com/tc39/test262">test262</a> for <i
...
     };
 
-    const niceEngineOrder = [ 'v8', 'v8_exp', 'sm', 'sm_exp', 'jsc', 'hermes', 'qjs', 'qjs_ng', 'libjs', 'chakra', 'graaljs', 'xs', 'rhino', 'nashorn', 'boa', 'kiesel', 'porffor', 'nova', 'engine262', 'babel', 'swc', 'sucrase' ];
+    const niceEngineOrder = [ 'v8', 'v8_exp', 'sm', 'sm_exp', 'jsc', 'hermes', 'qjs', 'qjs_ng', 'libjs', 'chakra', 'graaljs', 'xs', 'rhino', 'boa', 'kiesel', 'porffor', 'nova', 'engine262', 'babel', 'swc', 'sucrase' ];
 
     const thead = document.querySelector('thead');
     const tbody = document.querySelector('tbody');
 
-    let cwd = '', filterOutEngines = [ 'v8_exp', 'sm_exp', 'xs', 'rhino', 'nashorn', 'boa', 'qjs_ng', 'engine262', 'kiesel', 'porffor', 'nova', 'swc', 'sucrase' ], init = false;
+    let cwd = '', filterOutEngines = [ 'v8_exp', 'sm_exp', 'xs', 'rhino', 'boa', 'qjs_ng', 'engine262', 'kiesel', 'porffor', 'nova', 'swc', 'sucrase' ], init = false;
```
Comment (CanadaHonk) [inline site/index.html:140]: The `filterOutEngines.length !== 13` needs changing below too :sweat_smile:

**Review by CanadaHonk — CHANGES_REQUESTED** [review]

File: site/index.html (hunk 2 above)
Comment (linusg) [reply site/index.html:141]: :raised_eyebrow:

Done & left a comment :D

**Review by CanadaHonk — APPROVED** [review]
sorry but you've yee'd your last haw nashorn :gun::cowboy_hat_face:

**Pull request merged**